db/version_edit.cc

```diff
@@ -576,7 +576,7 @@ Status VersionEdit::DecodeFrom(const Slice& src) {
           return s;
         }
 
-        wal_deletions_.emplace_back(std::move(wal_deletion));
+        wal_deletion_ = wal_deletion;
```

Comment:
Any reason we can no longer move?

Reply:
we can still move, but it's the same as copy for now since it only holds a uint64_t.